ui/component/sideNavigation/view.jsx

```diff
@@ -377,6 +377,14 @@ function SideNavigation(props: Props) {
     return null;
   }
 
+  React.useEffect(
+    () => () => {
```

Comment:
Function that returns a function?

Reply:
It's a really weird syntax, but it is equivalent to componentWillUnmount() 
https://stackoverflow.com/questions/55020041/react-hooks-useeffect-cleanup-for-only-componentwillunmount

Reply:
Nice.  Btw, would adding a return statement to 
```
  React.useEffect(() => {
    // $FlowFixMe
    document.body.style.overflowY = showOverlay ? 'hidden' : '';
  }, [showOverlay]);
```
to handle the unmounting case be equivalent, or does it have to be an equivalent to `componentWillUnmount`?

Reply:
I think it can be done that way and I like that way more. I just didn't think to do that. Nice suggestion!